mbuild/formats/charmm_writer.py

```diff
@@ -1049,6 +1049,67 @@ def _lengths_angles_to_vectors(lengths, angles, precision=6):
     return box_vectors
 
 
+def _check_fixed_bonds_angles_lists(
+    fixed_residues_bonds_angles_list,
+    fixed_bonds_angles_residues_variable_name,
+    residues,
+):
+    """Check the GOMC fixed bonds and angles lists for input errors.
+
+    Parameters
+    ----------
+    fixed_residues_bonds_angles_list : list of strings, [str, ..., str]
+        A list of the residues (i.e., molecules since GOMC currently considers a
+        a whole molecule as a residue) to have their bonds and/or angles held
+        rigid/fixed for the GOMC simulation engine.
+        In GOMC, the residues currently the molecules.
+    fixed_bonds_angles_residues_variable_name : str
+        The name of the varible used as a string, which string, which will be
+        in the error and information outputs.
+    residues : list, [str, ..., str]
+        Labels of unique residues in the Compound. Residues are assigned by
+        checking against Compound.name.  Only supply residue names as 4 character
+        strings, as the residue names are truncated to 4 characters to fit in the
+        psf and pdb file.
+
+    Returns
+    -------
+    Provides a ValueError or TypeError if the input is not correct.
+    """
+
+    if fixed_residues_bonds_angles_list is not None and not isinstance(
+        fixed_residues_bonds_angles_list, list
+    ):
+        print_error_message = (
+            "ERROR: Please enter the residues names in the ({}) variable "
+            "are in a list format.".format(
+                fixed_bonds_angles_residues_variable_name
+            )
+        )
```

Comment:
I think this is meant to say:

```suggestion
        print_error_message = (
            "ERROR: Please ensure the residue names in the ({}) variable "
            "are in a list.".format(
                fixed_bonds_angles_residues_variable_name
            )
        )
```

Reply:
The first part of this comment should be addressed in @daico007 comments above.

The 2nd part is fixed as requested, with the changed :

"ERROR: Please ensure the residue names in the ({}) variable "
            "are in a list format.".format(